crates/nu-command/src/filesystem/open.rs

```diff
@@ -0,0 +1,53 @@
+use nu_protocol::ast::Call;
```

Comment:
For commands we aren't ready to implement, probably better to leave them out for now so we don't accidentally think they're done

Reply:
I'm reworking the `.map` now, so might have an easier way to do it in the future. Probably worth landing that part as is and then revisiting as more helpers are added.

Reply:
oh nice, yeah the issue now is that it will return a stream of errors as a value to the the client rather throwing any errors.  Is that alright in the meantime?